roles/cod_login_node/tasks/main.yml

```diff
@@ -117,8 +122,3 @@
   systemd:
```

Comment:
Can you move this task before login001 reboot on line 99 . tftp service should be up before reboot.

Reply:
After testing, the tftpd servic will be started when needed.